libsplinter/src/network/mod.rs

```diff
@@ -51,7 +51,7 @@ impl NetworkMessage {
     }
 }
 
-struct PeerMap {
+pub struct PeerMap {
```

Comment:
This feels like it should remain an implementation detail of the network layer, and if we expose it, we should consider if this is truly the right data structure across the API boundary. In any case, this does not differentiate between the types of "peers" (splinter<->splinter) vs (splinter<->service).